src/main/java/com/google/devtools/build/lib/starlarkbuildapi/repository/StarlarkRepositoryContextApi.java

```diff
@@ -329,8 +329,9 @@ void patch(Object patchFile, StarlarkInt strip, StarlarkThread thread)
       name = "download",
       doc =
           "Downloads a file to the output path for the provided url and returns a struct"
-              + " containing a hash of the file with the fields <code>sha256</code> and"
-              + " <code>integrity</code>.",
+              + " containing <code>success</code>, a flag which is <code>true</code> if the"
```

Comment:
Is it a "flag" when it is the status indicator, or is it "containing success, which is true if the download completed ...